### PR TITLE
CI: add retries to curl, update image and install helm from tar.gz

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

* Add retries to `curl` when installing dependencies in CI.
* Update travis image to `bionic`
* Install `helm` from `tar.gz` not via `snap` (way faster)

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+  - curl -Lo- https://get.helm.sh/helm-v2.16.12-linux-amd64.tar.gz | tar xvzf -
+  - sudo mv ./linux-amd64/helm /usr/local/bin/
+  # yq
+  - curl -L --remote-name https://github.com/mikefarah/yq/releases/download/3.2.1/yq_linux_amd64
+  - sudo mv yq_linux_amd64 /usr/local/bin/yq-3.2.1
   - sudo chmod +x /usr/local/bin/yq-3.2.1
   - sudo ln -s /usr/local/bin/yq-3.2.1 /usr/local/bin/yq
-  - curl -Lo- https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.x86_64.tar.xz | tar -xJf -
+  # shellcheck
+  - curl --retry 10 --retry-max-time 120 --retry-delay 5 -Lo- https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.x86_64.tar.xz | tar -xJf -
```
Comment (sumo-drosiek) [inline .travis.yml:20]: what was the issue? timeout?
what about other curl commands?

Comment (pmalek-sumo) [reply .travis.yml:20]: The linked issue was related to downloading `shellcheck` I can add retry parameters to other `curl` invocations as well

Comment (sumo-drosiek) [reply .travis.yml:20]: As we didn't investigate what is the reason we should do that, because e.g. yq is taken from the github as well

**Review by sumo-drosiek — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +1,24 @@
 lang: ruby
+# Somehow focal (20.04) is available for amd64 but when
+# ran on travis it falls back to xenial (16.04) so just use bionic (18.04)
```
Comment (perk-sumo) [inline .travis.yml:3]: Now, after we've migrated to travis-ci.com let's make sure the issue still exists.

**Review by perk-sumo — APPROVED** [review]
One nit, I'm good 👍

File: .travis.yml (hunk 2 above)
Comment (pmalek-sumo) [reply .travis.yml]: done

**Pull request merged**